src/rubicon/objc/eventloop.py

```diff
@@ -733,6 +767,25 @@ def set_child_watcher(self, watcher):
             self._watcher = watcher
 
 
+if sys.version_info < (3, 14):
+
+    def RubiconEventLoop():
+        """Create a new Rubicon CFEventLoop instance."""
+        # If they're using RubiconEventLoop(), they've done the necessary adaptation.
+        with warnings.catch_warnings():
+            warnings.filterwarnings(
+                "ignore",
+                message=r"^Custom EventLoopPolicy instances have been deprecated by Python 3.14",
+                category=DeprecationWarning,
+            )
+            policy = EventLoopPolicy()
+        set_event_loop_policy(policy)
+        return new_event_loop()
```

Comment:
I expect rubicon-objc to be typically used in cases where it's ok to set the event loop policy globally. However, it might be a bit less invasive and more compatible with the Python 3.14 approach if return the event loop instance without any side effects. For example:

```Python
policy = EventLoopPolicy()
return policy.get_event_loop()
```

WDYT?

Reply:
So... [`get_event_loop()` has been deprecated as well](https://github.com/python/cpython/issues/83710) (and has been for a couple of releases). 

The [Python PR that triggered this change](https://github.com/python/cpython/issues/127949) documents the "suggested" API usage as:
```
loop = MyCustomEventLoop()
loop.run_until_complete(task)
```

Reply:
> How about the following then?
> 
> ```
> policy = EventLoopPolicy()
> return policy.new_event_loop()
> ```

That's functionally identical to `asyncio.new_event_loop()` - but I guess it doesn't hurt to be explicit about which policy is creating the loop and avoiding a function call.
 
> This might be stupid question, but why special case Python < 3.14 at all? Could we use the "suggested" API usage and instantiate the `CFEventLoop` directly on older Python versions as well?

In < 3.14, there are still *some* functions on the policy - child watchers, mechanisms to get the default loop, and so on. For maximum compatibility, I figured it was best to preserve the old API until it is fully deprecated.